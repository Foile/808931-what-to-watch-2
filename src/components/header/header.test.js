import React from "react";
import renderer from "react-test-renderer";
import Header from "./header";
import {BrowserRouter} from "react-router-dom";

it(`Header correctly renders after relaunch`, () => {
  const tree = renderer
  .create(
      <BrowserRouter>
        <Header type="movie-card__head"
          auth = {{name: `test`, avatarUrl: `ya.ru`}} />
      </BrowserRouter>
  )
  .toJSON();
  expect(tree).toMatchSnapshot();
});
