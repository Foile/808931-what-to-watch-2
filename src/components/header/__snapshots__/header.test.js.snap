// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header correctly renders after relaunch 1`] = `
<header
  className="page-header movie-card__head"
>
  <div
    className="logo"
  >
    <a
      className="logo__link"
      href="/"
      onClick={[Function]}
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <div
    className="user-block"
  >
    <div
      className="user-block__avatar"
    >
      <a
        href="/mylist"
        onClick={[Function]}
      >
        <img
          alt="test"
          height="63"
          src="https://htmlacademy-react-2.appspot.com/ya.ru"
          width="63"
        />
      </a>
    </div>
  </div>
</header>
`;
